### DISCUSSION TITLE
MenuBar "root" short cut?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [tznind/TerminalGuiDesigner#206] [General]
Hi, i just started using TUI and TGD, i added a menu bar, the default File has a working F9 shortcut, but when i add another top level menu, i cannot find a way to add like F10 to this top level menu. I have tried  1 left click on the top level menu, pressing Ctrl+T, added F10, but then F10 got assigned to a child menu item instead.

I have looked in the MyView.cs and MyViewDesigner.cs and cannot find the F9 shortcut for the file menu and how its done.

**Comment by tznind** [comment]
Thanks for raising this issue.  F9 is the key to open the menu bar in general rather than a specific menu.  It is a setting in the main Terminal.Gui library.  I added the text to the File menu as a hint so users know what button to press.

I will have a look later and see if there is a way to to assign a shortcut to open the other menus but I think its not possible currently.

**Reply by GronbergInfo** [reply]
Thanks for the reply. Its probably not much to "work around" the stuff by "only" get to yse F9 to get th menu, i saw one can use short cuts in sub menus instead.
